### ISSUE TITLE
Issue with Webview -> when moving from one tab to another

### ISSUE THREAD

**Issue by skandasoft** [atom/atom#4744]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.165.0
**System**: Microsoft Windows 7 Home Premium 
**Thrown From**: Atom Core

### Stack Trace

Uncaught TypeError: undefined is not a function

```
At c:\Users\Administrator\AppData\Local\atom\app-0.165.0\resources\atom\renderer\lib\web-view.js:380

TypeError: undefined is not a function
  at WebView.handleBrowserPluginAttributeMutation (c:\Users\Administrator\AppData\Local\atom\app-0.165.0\resources\atom\renderer\lib\web-view.js:380:62)
  at browserplugin.proto.attributeChangedCallback (c:\Users\Administrator\AppData\Local\atom\app-0.165.0\resources\atom\renderer\lib\web-view.js:598:23)

```

### Commands

```
  4x -0:07.2 click (div.title)
     -0:02.3 did-start-loading (webview)
  2x -0:00.0 click (div.title)
```

### Config

```
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "seti-syntax"
    ]
  },
  "editor": {
    "fontSize": 1,
    "showInvisibles": true
  }
}
```

### Installed Packages

```coffee
# User
atom-runner, v2.1.1
block-travel, v0.10.0
change-case, v0.4.0
coffee-eval, v0.8.0
coffee-navigator, v0.0.14
coffee-refactor, v0.6.2
color-picker, v1.2.6
command-toolbar, v1.0.0
compare-files, v0.2.6
css-color-highlighting, v0.2.4
docblockr, v0.5.8
editor-stats, v0.16.0
emmet, v2.2.1
file-icons, v1.4.2
file-type-icons, v0.5.2
file-types, v0.3.0
filesize, v0.1.1
javascript-snippets, v1.0.0
language-ect, v0.2.0
language-jade, v0.3.0
language-slim, v0.3.0
language-stylus, v0.1.0
linter, v0.8.0
linter-coffeelint, v0.1.7
linter-csslint, v0.0.11
live-archive, v0.1.7
local-history, v2.0.0
open-last-project, v0.1.1
pain-split, v1.1.2
prefixr, v0.2.0
preview-plus, v0.0.0
project-manager, v1.14.0
refactor, v0.4.1
regex-railroad-diagram, v0.6.3
ruby-slim, v0.2.0
select-line, v0.16.0
set-syntax, v0.1.0
seti-syntax, v0.1.7
slim-language, v2.0.0
Stylus, v0.4.0
toggle-slash, v1.0.10
urlrequest, v0.1.0
web-browser, v0.2.3

# Dev
No dev packages
```

**Comment by mark-hahn** [comment]
FYI, this webview bug was fixed in version 0.177.0 if Atom.

**Comment by 50Wliu** [comment]
Thanks @mark-hahn.
